README.md

```diff
@@ -40,38 +40,6 @@ It is currently in active development in the Kubernetes community through the [d
 | Command                                              | Description                                                               | Demo|
 | ---------------------------------------------------- | --------------------------------------------------------------------------|-----|
 | [crio(8)](/docs/crio.8.md)                           | OCI Kubernetes Container Runtime daemon                                   ||
```

Comment:
I think we should leave a bread crumb somewerein here, even if it's just temporary (few months), saying that kpod has been moved to it's own project at "blah".